app/assets/javascripts/slices/app/helpers/i18n.js

```diff
@@ -0,0 +1,5 @@
+Handlebars.registerHelper('i18n', function(options) {
```

Comment:
Yeah I wasn't really sure what to do with this helper - it's a bit awkward, but it was the only way I (with @thomasmarshall's help) could get the `i18n` setting into the hbs templates. It would be great if we could think of a cleaner way of getting the `i18n` class applied to the correct inputs.

Well it could do what you said, @erino. That does actually seem better than returning the string. More flexible/useful in future situations.